practice/django_practice/employees/management/commands/initial_data_employee.py

```diff
@@ -0,0 +1,46 @@
+import random
+
+from faker import Faker
+from django.core.management.base import BaseCommand
+
+from employees.models import Employee
+from departments.models import Department
+
+
+class Command(BaseCommand):
+    help = "Create fake employees for testing"
+
+    def handle(self, *args, **kwargs):
+        self.stdout.write(self.style.SUCCESS("Creating fake employees..."))
+
+        departments = list(Department.objects.all())
+
+        employees = []
+
+        for i in range(200):
+            first_name = Faker().first_name()
+            last_name = Faker().last_name()
+            birthday = Faker().date_of_birth(minimum_age=20, maximum_age=60)
+            email = Faker().email()
```

Comment:
Using Enum instead, It useful when you change value for Enum gender without having to update code in many places.

Reply:
Done